app/src/main/res/layout/activity_confirm_request_leave.xml

```diff
@@ -30,309 +30,421 @@
+                                    >
+                                    <TextView
+                                        android:text="@string/to"
+                                        style="@style/StyleTitleConfirm"
+                                        />
+                                    <TextView
+                                        android:text="@{viewModel.request.compensation.toTime}"
+                                        style="@style/StyleTextConfirm"
+                                        />
+                                    <View
+                                        android:layout_width="match_parent"
+                                        android:layout_height="@dimen/dp_1"
+                                        android:layout_marginBottom="@dimen/dp_5"
+                                        android:layout_marginTop="@dimen/dp_5"
+                                        android:background="@color/colorPrimary"
+                                        />
+                                </LinearLayout>
+                            </TableRow>
 
-                        <TableRow
-                            android:gravity="center_vertical">
-                            <ImageView
-                                app:srcCompat="@drawable/ic_reason"
-                                tools:ignore="ContentDescription"
-                                style="@style/StyleIconConfirm"
-                                />
-                            <LinearLayout
-                                android:layout_marginLeft="@dimen/dp_20"
-                                android:layout_marginStart="@dimen/dp_20"
-                                android:orientation="vertical"
-                                >
-                                <TextView
-                                    android:text="@string/reason"
-                                    style="@style/StyleTitleConfirm"
+                            <TableRow
+                                android:gravity="center_vertical">
+                                <ImageView
+                                    app:srcCompat="@drawable/ic_reason"
+                                    tools:ignore="ContentDescription"
+                                    style="@style/StyleIconConfirm"
                                     />
-                                <TextView
-                                    style="@style/StyleTextConfirm"
-                                    />
-                            </LinearLayout>
-                        </TableRow>
-                    </TableLayout>
+                                <LinearLayout
+                                    android:layout_marginLeft="@dimen/dp_20"
+                                    android:layout_marginStart="@dimen/dp_20"
+                                    android:orientation="vertical"
+                                    >
+                                    <TextView
```

Comment:
WARNING: [TooManyViews] `activity_confirm_request_leave.xml` has more than 80 views, bad for performance
 Using too many views in a single layout is bad for performance. Consider using compound drawables or other tricks for reducing the number of views in this layout.

The maximum view count defaults to 80 but can be configured with the environment variable `ANDROID_LINT_MAX_VIEW_COUNT`.